src/api/Folder.js

```diff
@@ -315,7 +320,9 @@ class Folder extends Item {
                 },
             })
             .then(this.createSuccessHandler)
-            .catch(this.errorHandler);
+            .catch((e: $AxiosError<any>) => {
```

Comment:
What's the value of wrapping this in an anonymous function?

Reply:
I think this is leftover from a refactor of this where I was before passing the error callback to the errorHandler, but then changed it to be a class property